_docs/MAIN.md

```diff
@@ -1,3 +1,16 @@
 # terraform-aws-tardigrade-cloudwatch-log-metric-filter
 
 Terraform module to create CloudWatch Log Metric Filters
+
+# variable definitions
+
+As can be seen in the inputs, this module takes the variable `metric_filters`, which is a schema list consisting of the following fields:
+* name (string) - the name for the metric filter
+* filter_pattern (string) - CloudWatch Logs filter pattern for extracting metric data out of ingested log events
+* log_group_name (string) - The name of the log group to associate the metric filter with
+* metric_transformation (map) - A block defining collection of information needed to define how metric data gets emitted
+  - The metric_transformation map consists of the following fields:
```

Comment:
I think the description is cleaner if this sentence is part of the prior bullet. Then the sub-bullets can be moved up a level. Also, most markdown linters frown on mixing the bullet-style... e.g. `*` to `-`. Just use one or the other throughout.